### ISSUE TITLE
False positive: type mismatch?

### ISSUE THREAD

**Issue by swanson** [marcoroth/stimulus-lsp#28]
![image](https://github.com/marcoroth/stimulus-lsp/assets/56947/778c6176-44dd-4838-8015-8912115267b1)

```js
export default class extends Controller {
  static targets = ['container']
  static values = {
    selector: String,
    frameId: String,
    url: String,
    enabled: Boolean,
    border: Boolean,
  }
```

Is there some better way to write this so that the LSP will know that the ERB string interpolation is going to return a boolean? Otherwise, seems like this type mismatch is almost also going to fall for non String types?

**Comment by marcoroth** [comment]
Thanks for the report! 🙌🏼 

Yeah, I think this is similar to https://github.com/marcoroth/stimulus-lsp/issues/21.

We should probably just ignore any value that contains any form of `<%`, `<%=`, `<%-`, `%>`, or `-%>`

**Comment by marcoroth** [comment]
I just released 0.3.0 earlier today, which ships @Gert-JanPeeters' fix from #61. Would be curious to hear if that solves this issue, thank you!

**Comment by marcoroth** [comment]
Going to close this, feel free to re-open if you are still seeing something that doesn't seem right! Thank you!
